packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -270,6 +270,7 @@ class CupertinoTextField extends StatefulWidget {
     this.scrollPadding = const EdgeInsets.all(20.0),
     this.dragStartBehavior = DragStartBehavior.start,
     this.enableInteractiveSelection = true,
+    this.textSelectionControls,
```

Comment:
As you mentioned, yes we should change this to `selectionControls` in order to be consistent with the parameter in EditableText.  Good catch!